src/renderer/util.ts

```diff
@@ -85,7 +85,6 @@ export function forceUpdateElement(selector: string, all = false): void {
     all ? [...document.querySelectorAll(selector)] : [document.querySelector(selector)]
   ).filter(Boolean) as Element[];
 
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- May not actually have forceUpdate
```

Comment:
Why was this removed? Now there's an ESLint warning.

Reply:
by mistake ig, i forgor